recursive-merge

```diff
@@ -19,23 +21,27 @@ scc merge $MERGE_OPTIONS --shallow $BASE_BRANCH
 git submodule sync
 git submodule update --remote --recursive
 
-echo "Merge all submodules using repository configuration:"
-cat "scripts/repositories.yml"
-scc merge "--repository-config=repositories.yml" $MERGE_OPTIONS -S $STATUS --update-gitmodules --push $PUSH_BRANCH $BASE_BRANCH
+REPO_CONFIG=
+echo -n "Merge all submodules"
+if [ -f "scripts/repositories.yml" ]; then
```

Comment:
I'd propose moving this to the top-level in a follow up, unless we want to agree on some dotted directory.